<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graphing data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/graph.png</normaloff>../Images/graph.png</iconset>
  </property>
  <property name="locale">
   <locale language="Russian" country="Kazakhstan"/>
  </property>
  <widget class="QComboBox" name="comboBox_period">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0 - current </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>current-2 - current</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Personalized</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Periods">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Periods</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personalized_period">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>For example: 1-10, 8, 11-13</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>16</width>
     <height>471</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Companies">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Companies</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Company_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Parameters">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>30</y>
     <width>472</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>37</y>
     <width>20</width>
     <height>363</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Income_Statement">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Income Statement</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>240</y>
     <width>232</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Periods_Decisions">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decisions</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>160</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Periods_Production">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Production</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sales">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sales</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_COGS">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COGS (Cost of Goods)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Gross_Margin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gross Margin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Layoff_Charge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Layoff Charge</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Inventory_Charge">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inventory Charge</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Interest">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Profit_Before_Tax">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profit before Tax</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Tax">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tax</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Net_Profit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Net Profit</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Price">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Marketing">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marketing</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Production">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Production</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Investment">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Investment</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_RD">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R and D</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Production_CostUnit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Production Cost/Unit</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Capacity_Utilization">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capacity Utilization</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Inventory_units">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inventory (units)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Factory_Capacity">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Factory Capacity</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Employess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Employees</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Periods_Balance_Sheet">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance Sheet</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Cash">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cash</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Total_Assets">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Assets</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Inventory_cost">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inventory ($)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Loans">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loans</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Capital">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capital</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Liabilities_plus_Equity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Liabilities+Equity</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>290</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Periods_Marketing">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marketing</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sales_Made">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sales Made</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Unfilled_Orders">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unfilled Orders</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Total_CostUnit_Sold">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Cost/Unit Sold</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_MarginUnit_Sold">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Margin/Unit Sold</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>392</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_MPI">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>425</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MPI</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Show_Graph">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Graph</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
